extensions/amp-analytics/analytics-vendors-list.md

```diff
@@ -333,7 +339,7 @@ Adds support for Linkpulse. Configuration details can be found at [docs.linkpuls
 
 Type attribute value: `lotame`
 
-Adds support for Lotame. More information and configuration details can be found at [my.lotame.com](https://my.lotame.com/t/83h37h9/overview-1st-party-data-collection-in-google-amp).
+Adds support for Lotame. More information and configuration details can be found at `https://my.lotame.com/t/83h37h9/overview-1st-party-data-collection-in-google-amp`.
```

Comment:
why were these links changed?

Reply:
Those are dead links(404) and where blocking the PR. I addressed the issue in the comment above but haven't been getting any help. Any direction on how to proceed with the issues? Can you please check the above comment for additional details on the Percy test too..

Reply:
@erwinmombay